pkg/vdr/peer/vdr.go

```diff
@@ -50,5 +50,5 @@ func (v *VDR) Deactivate(did string, opts ...vdrapi.DIDMethodOption) error {
 
 // Accept did method.
 func (v *VDR) Accept(method string) bool {
-	return method == DIDMethod
+	return method == DIDMethod || method == "sov" // accept sov 'peer' did docs for interop
```

Comment:
@swcurran @andrewwhitehead @Moopli @llorllale I don’t understand this one. Why would a did:sov be interpreted as did:peer?

Reply:
If our peer DID docs are using did:sov then that's likely an error, although standardizing on did:peer also doesn't seem ideal until that spec is finalized.

Reply:
@Moopli @andrewwhitehead we can't treat did:sov as did:peer - if we are being sent did:sov, we must resolve it using the did:sov driver.

If this is happening, AcaPy must fix it.

Reply:
@Moopli please revert this change.

Reply:
@andrewwhitehead -- let us know the state of this.  If this convention is in the wild, how do we eliminate it without breaking the interop we have today and add the interop we need tomorrow.

Reply:
@swcurran  I'd start by just generating our current peer DID Doc implementation with the did:peer prefix to see how that works with interop (assuming we don't currently use did:peer, but I can't see any references to it in the code base). Maybe @sklump could have a look at that this week?

Reply:
@troyronda @andrewwhitehead I raised an issue over on aca-py for this https://github.com/hyperledger/aries-cloudagent-python/issues/1048

Reply:
Relevant: 

* https://github.com/hyperledger/aries-rfcs/issues/611
* https://github.com/hyperledger/aries-rfcs/issues/609